packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -714,7 +776,21 @@ export function useContest() {
     }
   }
 
+  useEffect(() => {
+    if (isPageProposalsLoading && sliceOfProposalsIdsToFetch.length > 0) {
+      const lastElementToLoadId = sliceOfProposalsIdsToFetch.slice(-1)[0];
+      const lastElementVotes = listProposalsData?.[lastElementToLoadId]?.votes;
+      const numberOfDataLoaded = sliceOfProposalsIdsToFetch.filter(id => {
+        return listProposalsData?.[id];
+      }).length;
+      if (lastElementVotes >= 0 && numberOfDataLoaded === sliceOfProposalsIdsToFetch.length) {
```

Comment:
Just out of curiousity - what is the `lastElementVotes >= 0` check here for??

Reply:
it's to control if the votes of the last proposal in the list were fetched, which is in turned used to hide the loader (via `setIsPageProposalsLoading(false)`)

Reply:
Cool, thank you!